src/task2/ReadWriteFile.java

```diff
@@ -0,0 +1,40 @@
+package task2;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReadWriteFile {
+    public static void main(String[] args) throws IOException {
```

Comment:
do not throw from main, try to handle exceptions here

Reply:
ok thank you.